src/libraries/System.Private.Xml/src/System/Xml/Core/XmlReader.cs

```diff
@@ -140,7 +140,7 @@ public virtual bool HasValue
         public abstract int Depth { get; }
 
         // Gets the base URI of the current node.
-        public abstract string? BaseURI { get; }
```

Comment:
Doesn't the ref need to be updated as well?

Reply:
(Note as well this needs to be added to the breaking change list for nullable annotations.)